docker-images/kafka/scripts/kafka_run.sh

```diff
@@ -11,7 +11,7 @@ export KAFKA_BROKER_ID=$(hostname | awk -F'-' '{print $NF}')
 echo "KAFKA_BROKER_ID=$KAFKA_BROKER_ID"
 
 # create data dir
-export KAFKA_LOG_DIRS=$KAFKA_VOLUME$KAFKA_LOG_BASE_NAME$KAFKA_BROKER_ID
+#export KAFKA_LOG_DIRS=$KAFKA_VOLUME$KAFKA_LOG_BASE_NAME$KAFKA_BROKER_ID
```

Comment:
remove, don't comment

Reply:
I forgot about that. I am also removing some other env var we don't need in the script anymore.